sensor/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,118 @@
+| `lidartag_process_noise_rotation_dot`         | `double`              | `0.1`                            | The square of this value in degree/second is part of input for Kalman Filter to process noise covariance matrix (Q).                                      |
+| `apriltag_max_convergence_translation`        | `double`              | `2.0`                            | Maximum translation error in centimeters allowed for an AprilTag detection hypothesis to be considered as converged.                                      |
+| `apriltag_new_hypothesis_translation`         | `double`              | `20.0`                           | Translation threshold in centimeters for generating a new hypothesis in AprilTag tracking.                                                                |
+| `apriltag_measurement_noise_translation`      | `double`              | `0.2`                            | The square of this value in meters is part of input for Kalman Filter to measurement noise covariance matrix (R).                                         |
+| `apriltag_process_noise_translation`          | `double`              | `0.02`                           | The square of this value in meters is part of input for Kalman Filter to process noise covariance matrix (Q).                                             |
+
+## Requirements
+
+### LiDARTag
+
+To perform camera-lidar calibration using this tool, it is necessary to prepare LidarTags and lidars with intensity measures. To ensure that no objects obstruct the tag detection and to achieve the most stable detection possible, it is highly recommended to also prepare fixed mounts for these tags, as shown below.
+
+Note that the ones we used are Lidartags of size 0.8 meters. Meaning the Apriltag payload is 0.6 meters. We have also tried with Lidartags of sizes 0.6 meters, but to use them the user needs to set several parameters by himself.
+
+<p align="center">
+    <img src="../docs/images/tag_based_pnp_calibrator/lidartag-mount.jpg"  alt="lidartag-mount" width="500">
+</p>
+
+## References
+
+References/External links
+[1] Jiunn-Kai (Bruce) Huang, Shoutian Wang, Maani Ghaffari, and Jessy W. Grizzle, "LiDARTag: A Real-Time Fiducial Tag System for Point Clouds," in IEEE Robotics and Automation Letters. Volume: 6, Issue: 3, July 2021.
+
+## Known issues/limitations
+
+- The tool uses a basic OpenCV camera model for calibration (plumb bomb).
+
+## Pro tips/recommendations
+
+- During calibration, ensure that the lidar scan covers the tag, similar to the first example shown in the image below. However, if the tag resolution is low, as in the second example, and the lidar still detects the tag, it may still be acceptable, but avoidable when possible. The third example demonstrates a scenario where the lidar scan fails to cover the tag, resulting in the inability to detect the LidarTag.
```

Comment:
```suggestion
- During calibration, ensure that the lidar scan covers the tag, similar to the first example shown in the image below. However, if the tag resolution is low, as in the second example, and the lidar still detects the tag, it may still be acceptable, but should be avoided when possible. The third example demonstrates a scenario where the lidar scan fails to cover the tag, resulting in the inability to detect the LidarTag.
```

Reply:
Thanks! fixed in (https://github.com/tier4/CalibrationTools/pull/164/commits/2dee0c576e69b31970d190961530dc63acb91779)